tensorflow_lite_support/c/task/vision/image_classifier.cc

```diff
@@ -167,8 +175,25 @@ TfLiteClassificationResult* GetClassificationResultCStruct(
 
 TfLiteClassificationResult* TfLiteImageClassifierClassifyWithRoi(
     const TfLiteImageClassifier* classifier,
-    const TfLiteFrameBuffer* frame_buffer, const TfLiteBoundingBox* roi) {
-  if (classifier == nullptr || frame_buffer == nullptr) {
+    const TfLiteFrameBuffer* frame_buffer, const TfLiteBoundingBox* roi,
+    TfLiteSupportError** error) {
+  if (classifier == nullptr) {
+    tflite::support::CreateTfLiteSupportError(
+        kInvalidArgumentError, "Expected non null image classifier.", error);
+    return nullptr;
+  }
+
+  if (frame_buffer == nullptr) {
+    ::tflite::support::CreateTfLiteSupportError(
+        kInvalidArgumentError, "Expected non null frame buffer.", error);
+    return nullptr;
+  }
+
+  StatusOr<std::unique_ptr<FrameBufferCpp>> cpp_frame_buffer_status =
+      ::tflite::task::vision::CreateCppFrameBuffer(*frame_buffer);
```

Comment:
I agree to change CreateCppFrameBuffer and allow it to load a pointer instead of a reference. frame_buffer is created as a pointer;  it's more natural to pass it into CreateCppFrameBuffer directly.

Reply:
Fixed